Author: Dererk <dererk@debian.org> 
Description: Remark on ntpd.conf(5) manpage that Debian does not bind to any address
by default.
Bug-Debian: 575705

diff --git a/ntpd.conf.5 b/ntpd.conf.5
index 339685d..d57098e 100644
--- a/ntpd.conf.5
+++ b/ntpd.conf.5
@@ -47,9 +47,11 @@ Keywords may be specified multiple times within the configuration file.
 They are as follows:
 .Bl -tag -width Ds
 .It Ic listen on Ar address
+OpenNTPd has the ability to sync the local clock to remote NTP servers and, if this directive is specified, can act as NTP server itself, redistributing the local clock.
+.Pp
 Specify a local IP address or a hostname the
 .Xr ntpd 8
-daemon should listen on.
+daemon should listen on to enable remote clients synchronization.
 If it appears multiple times,
 .Xr ntpd 8
 will listen on each given address.
